router/internal/requestlogger/access_log_expression.go

```diff
@@ -0,0 +1,32 @@
+package requestlogger
+
+import (
+	"fmt"
+	"github.com/expr-lang/expr/vm"
+	exprlocal "github.com/wundergraph/cosmo/router/internal/expr"
+	"github.com/wundergraph/cosmo/router/pkg/config"
+)
+
+type ExpressionAttribute struct {
+	Key     string
+	Default string
+	Expr    *vm.Program
+}
+
+func GetAccessLogConfigExpressions(attributes []config.CustomAttribute) ([]ExpressionAttribute, error) {
+	exprSlice := make([]ExpressionAttribute, 0)
+	for _, sAttribute := range attributes {
+		if expr := sAttribute.ValueFrom.Expression; expr != "" {
+			expression, err := exprlocal.CompileAnyExpression(expr)
```

Comment:
We should evaluate if the expression returns a loggable value. Is it possible to return no value like undefined? There is a checker module `github.com/expr-lang/expr@v1.16.9/checker` that can be used to check the return type of an expression.